### ISSUE TITLE
Secondary actions for list controls are buggy in demo

### ISSUE THREAD

**Issue by boydkr** [miguelcobain/ember-paper#392]
Secondary actions hold focus, and move around after being clicked.  I was looking at using `ember-paper` for some button lists w/ actions in my app, but it doesn't seem like it's in a usable state right now.

http://miguelcobain.github.io/ember-paper/release-0-2/#/list-controls

I am using Chrome on OSX

**Comment by miguelcobain** [comment]
@boydkr i see you're referencing the 0.2.x docs. Has this issue been fixed in master for you?

**Comment by boydkr** [comment]
I'm using 0.2.10, I didn't realize there were doc pages for newer versions.

I tried the docs for `release-1`, and I see that those issues appear to be fixed.

I noticed two other issues on the `release-1` demo page:
 * Checkboxes are not checked if you click directly on the box.
 * The "Avatar with Secondary Action icon" demo will trigger both the primary and secondary actions when you click the secondary icon
